packages/wp-dashboard/src/components/editorSettings/customFonts/index.js

```diff
@@ -242,6 +258,32 @@ function CustomFontsSettings({
     [handleOnSave]
   );
 
+  useEffect(() => {
+    const el = currentFontsRowsRef.current[`row-${currentFontsFocusIndex}`];
```

Comment:
What is happening here?

Reply:
That's a ref back to the row that needs to get focus

The new index is set here:

```
if (key === 'ArrowUp') {
        evt.preventDefault();
        setCurrentFontsFocusIndex((index) => Math.max(0, index - 1));
      } else if (key === 'ArrowDown') {
        evt.preventDefault();
        setCurrentFontsFocusIndex((index) =>
          Math.min(customFonts.length - 1, index + 1)
        );
      }
```

Reply:
Added some comments per the other comment below